File: CSharp/Supermarket/Model/Checkout/Receipt.cs
using System.Collections.Generic;

namespace Supermarket.Model.Checkout
{
    public class Receipt
    {
        public Receipt(List<BookingLine> bookingLines, List<TaxLine> taxLines, decimal total)
        {
            Bookings = bookingLines;
            TaxLines = taxLines;
            Total = total;
        }

        public List<BookingLine> Bookings { get; }
        public List<TaxLine> TaxLines { get; }
        public decimal Total { get; }
    }
}

File: java/src/main/java/supermarket/model/checkout/Receipt.java
package supermarket.model.checkout;

import java.math.BigDecimal;
import java.util.List;

public class Receipt {
	private List<BookingLine> bookings;
	private List<TaxLine> taxLines;
	private BigDecimal total;

	public List<BookingLine> getBookings() {
		return bookings;
	}

	public void setBookings(List<BookingLine> bookings) {
		this.bookings = bookings;
	}

	public List<TaxLine> getTaxLines() {
		return taxLines;
	}

	public void setTaxLines(List<TaxLine> taxLines) {
		this.taxLines = taxLines;
	}

	public BigDecimal getTotal() {
		return total;
	}

	public void setTotal(BigDecimal total) {
		this.total = total;
	}
}
